go-bindata-assetfs/main.go

```diff
@@ -29,6 +29,13 @@ func isDebug(args []string) bool {
 }
 
 func main() {
+	debug := isDebug(os.Args[1:])
```

Comment:
Thanks for the PR.
Why won't you put that in a function, like the isDebug parameter?